playbooks/bootstrap_ml2_vlan.yml

```diff
@@ -36,6 +36,11 @@
     - ucarp
   - apt: upgrade=dist
 
+  - name: load vlan module
+    lineinfile: dest=/etc/modules
+                line="8021q"
+    when: ansible_distribution_version == "14.04"
```

Comment:
Is this necessary? I haven't had a problem creating a VLAN interface without it.